File: src/test/java/com/antero/liemipeli/domain/VampyyriTest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.antero.liemipeli.domain;

import org.junit.Before;
import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author Antero Oikkonen
 */
public class VampyyriTest {
    
    Vampyyri v = new Vampyyri();
    
    public VampyyriTest() {
    }
    
    @Before
    public void setUp() {
    }

    @Test
    public void testSetNimi() {
        v.setNimi("Darren");
        assertEquals("Darren", v.getNimi());
    }
    
}

File: liemipeli-kotlin/src/test/kotlin/com/antero/liemipeli/domain/VampyyriTest.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.antero.liemipeli.domain

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

/**
 *
 * @author Antero Oikkonen
 */
class VampyyriTest {

    internal var v = Vampyyri("Darren")

    @Test
    fun testSetNimi() {
        assertEquals("Darren", v.nimi)
    }

}
